framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/OpenShiftScenario.java

```diff
@@ -236,4 +242,12 @@ public void afterScenarioFinished(T deploymentScenario) {
 
         });
     }
+
+    @Override
+    public Optional<MavenRepositoryDeployment> getMavenRepositoryDeployment() {
```

Comment:
If we try to access the Maven Repository Deployment and there is none. That would mean there is no Kjar deployment ...
Shouldn't we raise an error instead ?

Reply:
Well, that is a question.
I think it depends on the fact if the information about whether Maven repository is deployed is relevant to user. If user calls this method to figure out whether the Maven repository is deployed or rather just want to get it.

Well, for our case I suppose that when user calls this method then he needs to use Maven repository. Will adjust the implementation to throw an exception.